src/shared/api/CancerHotspotsAPI.ts

```diff
@@ -0,0 +1,1044 @@
+import * as request from "superagent";
+
+type CallbackHandler = (err: any, res ? : request.Response) => void;
+export type HotspotMutation = any;
+export type SingleResidueHotspotMutation = any;
+export type IntegerRange = any;
+export type TumorTypeComposition = any;
+export type Cluster = any;
+export type ClusteredHotspotMutation = any;
```

Comment:
I checked the swagger doc and it looks like there are some errors in it.  could be why the swagger generator fails even though the types do seem like the are specified.